data/scripting/messages.lua

```diff
@@ -67,13 +80,16 @@ end
 --    e.g. positioning of message boxes.
 --
 --    :arg player: the recipient of the message
+--    :type player: :class:`wl.game.Player`
 --    :arg title: the localized title of the message
 --    :type title: :class:`string`
 --    :arg message: the localized body of the message. You must use
---                 :ref:`richtext functions <richtext.lua>` here.
+--                 :ref:`richtext functions <richtext.lua>` here,
+--                 e.g. ``p(_"message")``
```

Comment:
same here, and more below

Reply:
Done, also changed this in the code part. Also for `win_conditions/win_condition_texts.lua`